src/main/java/org/testng/reporters/JUnitReportReporter.java

```diff
@@ -81,12 +82,14 @@ public void generateReport(List<XmlSuite> xmlSuites, List<ISuite> suites,
 //      afters.put(es.getKey(), es.getValue().iterator());
 //    }
 
+    SimpleDateFormat df = new SimpleDateFormat("dd MMM yyyy HH:mm:ss");
     for (Map.Entry<Class<?>, Set<ITestResult>> entry : results.entrySet()) {
       Class<?> cls = entry.getKey();
       Properties p1 = new Properties();
       p1.setProperty("name", cls.getName());
       Date timeStamp = Calendar.getInstance().getTime();
-      p1.setProperty(XMLConstants.ATTR_TIMESTAMP, timeStamp.toGMTString());
+      String timestampStr = df.format(timeStamp) + " GMT";
```

Comment:
The goal of this reporter is to generated a JUnit XML report file. In fact, this report file was never generated by JUnit, but by the junit task of ant.
Have a look to [the usage of timestamp](https://fisheye6.atlassian.com/browse/~br=master/ant-git/src/main/org/apache/tools/ant/taskdefs/optional/junit/XMLJUnitResultFormatter.java?r=bf6026a147e879837eaad2a16348ae2162829867&r=df33426bf23dd97171dc3e0e7cb85de87d421bce#to138), and to [its expected format](https://fisheye6.atlassian.com/browse/~br=master/ant-git/src/main/org/apache/tools/ant/util/DateUtils.java?r=bf6026a147e879837eaad2a16348ae2162829867&r=df33426bf23dd97171dc3e0e7cb85de87d421bce&r=43844a7e6b54e2dd27216b30f6d7b44bb601d29c#to49).